modular_citadel/code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -53,6 +53,7 @@
 	has_snowflake_deadsprite = TRUE
 	dogborg = TRUE
 	cyborg_pixel_offset = -16
+	borghealth = 80
```

Comment:
Why the K9 unit susceptible to being completely fucking disabled with one shotgun blast? or 357 shot or 3 shots from blasters?

The fuck kind of balance is that.